### ISSUE TITLE
[Feature Request] Adding Ollama support

### ISSUE THREAD

**Issue by Andurilll** [chriskoch/ngx-renamer#3]
I would love to test this post-consumer script but i am worried about sending my private data to openai.

Ollama support would solve this issue. Maybe this makes it possible to change between ollama & openai:
https://ollama.com/blog/openai-compatibility

**Comment by chriskoch** [comment]
Yes, it already runs on Ollama in a local secure environment, I am trying to find some time to push it to github.com, soon. Still have issues finding the best model and prompt.

**Comment by kennyluk** [comment]
I have forked the original `ngx-renamer` project and made modifications to get it working with a local copy of ollama. You can use this as a reference for your own implementations. I want to note that I had never written a single line of Python before, and I was thoroughly helped by Microsoft Copilot to achieve this. Use it as-is.
